README.rst

```diff
@@ -102,24 +102,67 @@ Development
 
 Follow the `CKAN style rules <http://docs.ckan.org/en/latest/contributing/css.html#formatting>`_.
 
-Converting to Less for styling.
-
-This is the current process until a cleaner setup can be created.
-
-Install npm and less, then compile less files to css before pushing changes.::
-
-    sudo apt install npm
-    sudo npm install -g less
-    ln -s /usr/bin/nodejs /usr/bin/node
-    cd /ckanext-ontario_theme/ckanext/ontario_theme/fanstatic/internal
-    lessc ontario_theme.less ontario_theme.css # Builds internal
-    cd ../external
-    lessc ontario_theme.less ontario_theme.css # Builds external
-
-Styles should be broken down into small modules that do one thing and contain all necessary 
-styling for that module. As an example, the smarties.less file should contain all styling
-needed for smarties.
-
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Installing the pre-commit hooks for development
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Activate your CKAN virtualenv and run:
+::
+ pip install -r dev-requirements.txt
+ pre-commit install
+
+We use the djLint pre-commit hook to lint our code.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Reformatting code or linting code outside of the pre-commit hooks
```

Comment:
lines 117-139 are not needed because we are using pre-commit hooks, which already uses a different environment than the ckan-dev env already.

for linting outside of the pre-commit-hooks, or outside of the env, isn't there a way to trigger the pre-commit-hooks command without doing a commit? that will be much simpler to do and document.

Reply:
We can pass arguments to hooks. I'll remove this section and look into adding reformatting arguments. For now it's best to leave it on confluence